packages/analytics-js-integrations/src/integrations/GA4/utils.js

```diff
@@ -24,6 +24,35 @@ const extractLastKey = key => key.split('.').pop();
  */
 const shouldSendUserId = integrations => integrations?.GA4?.sendUserId ?? true;
 
+/**
+ * Returns user traits
+ * @param {*} piiPropertiesToIgnore 
+ * @param {*} userTraits 
+ * @returns 
+ */
+const filterUserProperties = (piiPropertiesToIgnore, userTraits) => {
```

Comment:
Let's call it `filterUserTraits` and update the nomenclature in other variables also.

Reply:
Please add more descriptive JS doc

Reply:
updated